docs/api.rst

```diff
@@ -4470,6 +4470,15 @@ Guild
 
         :type: :class:`User`
 
+GuildBuilder
+~~~~~~~~~~~~~
+
+.. attributetable:: GuildBuilder
+
+.. autoclass:: GuildBuilder()
+    :members:
+    :exclude-members: add_category_channel
```

Comment:
Why is `add_category_channel` excluded?

Reply:
It's an alias of `add_category` - since sphinx can't properly detect method aliases, it'd otherwise show the entire docstring twice :/

Reply:
Perhaps, but it'll look out of place as sphinx wouldn't put it next to `add_category` in the docs, but instead all the way at the top of the method list. I've added a mention of `add_category_channel` to the `add_category` docs now, that should do it as well.
91ce8dd8d621be24bf114f590dee96aa98291579